addon/model.js

```diff
@@ -472,7 +484,6 @@ MegamorphicModel.prototype.store = null;
 MegamorphicModel.prototype._internalModel = null;
 MegamorphicModel.prototype._parentModel = null;
 MegamorphicModel.prototype._topModel = null;
-MegamorphicModel.prototype.id = null;
```

Comment:
Why drop this from the prototype? Isn't that just going to get us down the `setUnknownProperty` path unnecessarily?

Reply:
It won't because `id` is now a class property.